### ISSUE TITLE
Feature Request: Light Mode for Base Bridge

### ISSUE THREAD

**Issue by stevenmalenz** [base-org/web#450]
### Is your feature request related to a problem? Please describe.

There are a few accessibility and usability issues with the Base Bridge website. A few are color contrast related. And usability wise, the website is current a permanently 'dark mode' website. 

### Describe the solution you'd like

Would love to work with @carlaiau to provide the option to switch to a 'light mode' to improve the user experience for those who prefer it, and also the accessibility.

### Describe alternatives you've considered

Short-term alternatives are around increasing the colour contrast of a lot of the darker grey elements on the black background, like the chain selected.

**Comment by wbnns** [comment]
@stevenmalenz 

Heya! Thanks for this suggestion. We are actually going to be deprecating the bridge soon, as we further decentralize. Details here: https://github.com/base-org/web/pull/479

Otherwise personally agree with you that this would be helpful!
